src/App.tsx

```diff
@@ -1,61 +1,116 @@
+import { useState } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { Todo } from './types/Todo';
+
+import { TodoList } from './components/TodoList';
+
+const getUserById = (id: number) => {
+  return usersFromServer.find(user => user.id === id) || null;
+};
+
+const todosWithUser: Todo[] = todosFromServer.map(todo => ({
+  ...todo,
+  user: getUserById(todo.userId),
+}));
+
+const generateNewId = (todos: Todo[]) => {
```

Comment:
Consider using `Math.max(...todos.map(todo => todo.id)) + 1` instead of `maxId + 1` for `generateNewId` to make it more concise. 🤔